app/views/how-tos/github/collaborate.html

```diff
@@ -0,0 +1,130 @@
+{% extends "layout.html" %} {% block pageTitle %} Collaborate on prototypes
+using GitHub Desktop - NHS prototype kit {% endblock %} {% block beforeContent
+%} {{ breadcrumb({ items: [ { href: "/", text: "Home" }, { href: "/how-tos",
+text: "Guides" }, { href: "/how-tos/github/index", text: "Store code online with
+GitHub and GitHub desktop" } ] }) }} {% endblock %} {% block content %}
```

Comment:
Could this get spread out? it's quite hard to read.

Reply:
gah this was prettifier not dealing well with nunjucks and me being lazy. fixed